src/main/java/com/google/sps/data/OrganizationUpdater.java

```diff
@@ -44,7 +44,12 @@ public void updateOrganization(HttpServletRequest request, GivrUser user, boolea
     Set<String> requiresModerator = new HashSet<String>();
     Map<String, String> properties = new HashMap<String,String>();
     boolean isMaintainer = user.isMaintainer();
-    boolean isModerator = user.isModerator(this.entity.getKey().getId());
+    boolean isModerator = false;
+
+    // Gets moderator status from checking its moderatingOrgs size.
+    if (user.getModeratingOrgs().size() > 0) {
+      isModerator = true;
+    }
```

Comment:
Could you encapsulate this logic in user? What if you had a function: User.IsModeratorOfAnyOrg?